ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/ChatCommandModal.qml

```diff
@@ -0,0 +1,195 @@
+    property string finalButtonLabel: "Request address"
+    property var sendChatCommand: function () {}
+    property bool isRequested: false
+
+    id: root
+    title: root.commandTitle
+    height: 504
+
+    property var selectedRecipient
+    onSelectedRecipientChanged: {
+        selectRecipient.selectedRecipient = this.selectedRecipient
+        selectRecipient.readOnly = !!this.selectedRecipient && !!this.selectedRecipient.address
+    }
+
+    onClosed: {
+        stack.reset()
+    }
+
+    TransactionStackView {
+        id: stack
+        anchors.fill: parent
+        anchors.leftMargin: Style.current.padding
+        anchors.rightMargin: Style.current.padding
+        onGroupActivated: {
+            root.title = group.headerText
+            btnNext.label = group.footerText
+        }
+        TransactionFormGroup {
+            id: group1
+            headerText: root.commandTitle
+            footerText: qsTr("Continue")
+
+            AccountSelector {
+                id: selectFromAccount
+                accounts: walletModel.accounts
+                selectedAccount: walletModel.currentAccount
+                currency: walletModel.defaultCurrency
+                width: stack.width
+                label: qsTr("From account")
+                reset: function() {
+                    accounts = Qt.binding(function() { return walletModel.accounts })
+                    selectedAccount = Qt.binding(function() { return walletModel.currentAccount })
+                }
+            }
+            SeparatorWithIcon {
+                id: separator
+                anchors.top: selectFromAccount.bottom
+                anchors.topMargin: 19
+            }
+            RecipientSelector {
+                id: selectRecipient
+                accounts: walletModel.accounts
+                contacts: profileModel.addedContacts
+                label: qsTr("Recipient")
+                readOnly: true
+                selectedRecipient: root.selectedRecipient
+                anchors.top: separator.bottom
+                anchors.topMargin: 10
+                width: stack.width
+                reset: function() {
+                    isValid = true
+                }
+            }
+        }
+        TransactionFormGroup {
+            id: group2
+            headerText: root.commandTitle
+            footerText: qsTr("Preview")
+
+            AssetAndAmountInput {
+                id: txtAmount
+                selectedAccount: selectFromAccount.selectedAccount
+                defaultCurrency: walletModel.defaultCurrency
+                getFiatValue: walletModel.getFiatValue
+                getCryptoValue: walletModel.getCryptoValue
+                isRequested: root.isRequested
+                width: stack.width
+                reset: function() {
+                    selectedAccount = Qt.binding(function() { return selectFromAccount.selectedAccount })
+                }
+            }
```

Comment:
Adding a `GasValidator` component here would ensure the user gets feedback when they do not have enough ETH for gas (there is logic in that component that checks if they are sending ETH, then adds the tx ETH to the gas ETH). Also, adding an invisible `GasSelector` component aids in ensuring we have correct gas values (and could be made visible in the future to align with the design). This was done in the StickerPack purchase modal:

https://github.com/status-im/nim-status-client/blob/dff022bac088cd3f6f89e890e3c712c11fe33f29/ui/app/AppLayouts/Chat/components/StickerPackPurchaseModal.qml#L102-L138

However, please use the `gasEstimate` recommendation in the already merged PR (https://github.com/status-im/nim-status-client/pull/822#discussion_r485583885)

Reply:
I won't add it here since it's just the request for a transaction. There is no signing any time soon, so I assume that if the user has enough Token to send to the other, he probably has enough for gas. Anyway, he can acquire it in the meantime